boilerplate/openshift/osd-container-image/project.mk

```diff
@@ -0,0 +1,9 @@
+IMAGE_REGISTRY?=quay.io
+IMAGE_REPOSITORY?=app-sre
+IMAGE_NAME?=osd-cluster-ready
```

Comment:
Ah good catch - I settled on `IMAGE_NAME` because the convention is for container images, which lines up nicely with the other `IMAGE_*` variables.

I think omitting it entirely makes the most sense to minimize unintentional bootstrapping errors, e.g. for repos like splunk-forwarder-images.

Reply:
Cool. I see you've got a check in standard.mk to ensure it's defined, so ✓